From 0121c5d9647a0b0a79e27451325b38a732f03f9d Mon Sep 17 00:00:00 2001
From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 13:13:30 -0700
Subject: use a domain that is more likely invalid

 avoids test failures in unisolated builds
Patch-Name: domain-test.patch
---
 tornado/test/netutil_test.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tornado/test/netutil_test.py b/tornado/test/netutil_test.py
index 7d9cad3..6206341 100644
--- a/tornado/test/netutil_test.py
+++ b/tornado/test/netutil_test.py
@@ -56,7 +56,7 @@ class _ResolverErrorTestMixin(object):
             return True  # Halt propagation.
 
         with ExceptionStackContext(handler):
-            self.resolver.resolve('an invalid domain', 80, callback=self.stop)
+            self.resolver.resolve('anasdas23dadasdasdasdi2n1validsdasdbwefrwfejnefsegfsdomain', 80, callback=self.stop)
 
         result = self.wait()
         self.assertIsInstance(result, Exception)
@@ -64,7 +64,7 @@ class _ResolverErrorTestMixin(object):
     @gen_test
     def test_future_interface_bad_host(self):
         with self.assertRaises(Exception):
-            yield self.resolver.resolve('an invalid domain', 80,
+            yield self.resolver.resolve('ana8sdasdadasdasdas1d12invalidsdasdbwefrwfejnefsegfsdomain', 80,
                                         socket.AF_UNSPEC)
 
 
